02.calendar/cal.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'date'
+
+opt = OptionParser.new
+opt.on('-m')
+opt.on('-y')
+
+opt.parse!(ARGV)
+month = ARGV[0] || Date.today.mon
+year = ARGV[1] || Date.today.year
+
+header = "#{month}月"+" "+year.to_s
+print header.center(20) + "\n"
```

Comment:
[nits] わざわざ変数にいれなくても出力できそうですね。
改行ありで出力するなら `puts` を使ってよいと思います。

```suggestion
puts "#{month}月 #{year}".center(20)
```